libraries/Wire/src/utility/twi.c

```diff
@@ -396,7 +418,39 @@ void twi_releaseBus(void)
   twi_state = TWI_READY;
 }
 
-ISR(TWI_vect)
+/*
+ * Function twi_timeout
+ * Desc     Increases the counter in a loop
+ *			and if it reaches a maximum value, 
+			the hardware is initialized and 
+			the function returns true 
+ * Input    isInitializationRequired: If 1 the 
+			timer resets the counter
+ * Output   1 Timeout occurred          
+ *          0 No timeout occurred
+*/
+
+uint8_t twi_timeout(uint8_t isInitializationRequired)
+{
+	if (isInitializationRequired) 
+	{
+		// reset the timeout counter
+		twi_timer_count=0;
+	}
+	else 
+	{
```

Comment:
Please follow the formatting style of the rest of the code in the file and attach the brace to the `else` statement.